azure/packages/azure-client/src/index.ts

```diff
@@ -24,10 +24,11 @@ export type {
 	AzureRemoteConnectionConfig,
 	AzureUser,
 	IAzureAudience,
-	ITelemetryBaseEvent,
-	ITelemetryBaseLogger,
 } from "./interfaces";
 
 export type { ITokenProvider, ITokenResponse } from "@fluidframework/routerlicious-driver";
 export type { ITokenClaims, IUser } from "@fluidframework/protocol-definitions";
 export { ScopeType } from "@fluidframework/protocol-definitions";
+
+// Re-export so developers can build loggers without pulling in core-interfaces
+export type { ITelemetryBaseEvent, ITelemetryBaseLogger } from "@fluidframework/core-interfaces";
```

Comment:
i don't know if it works, but maybe if you add a file, like exports.ts. that file re-exports from all the things. in the package we import from that file, and in the index we export again

Reply:
This change is a no-op for the consumer. We're already exporting these types. And since this package depends on `core-interfaces`, the end user doesn't have to import that package directly to get access to these types, they can just do:
```typescript
import { ITelemetryBaseEvent } from "@fluidframework/azure-client";
```

I'm not arguing for or against the re-exports here, to be clear, but these changes are strictly trying to make the existing API pattern a bit more discoverable (since we were both confused by this while testing release tags).

Reply:
yeah. i'm not blocking, i'm just trying to understand how a consumer know where to get the types from. ideally core-interfaces wouldn't even show up anywhere outside the code, and it would look like they just come from this package, not sure that is possible.